subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -349,12 +349,12 @@ static int do_deregister_reply_cb(const struct coap_packet *response,
 	u8_t code;
 
 	code = coap_header_get_code(response);
-	LOG_DBG("Deregister callback (code:%u.%u)",
+	LOG_INF("Deregister callback (code:%u.%u)",
 		COAP_RESPONSE_CODE_CLASS(code),
 		COAP_RESPONSE_CODE_DETAIL(code));
 
 	if (code == COAP_RESPONSE_CODE_DELETED) {
-		LOG_DBG("Deregistration success");
+		LOG_INF("Deregistration success");
```

Comment:
same here

Reply:
Thanks for the comment. I'm thinking that it'll be good to have de-register logging at the same level as register (as in "do_registration_reply_cb()"). App developer does not want debug prints but would be nice to know the de-register result.

Reply:
Sure, if LOG_INF() works here and does not print too much data, then it is ok. It is just that usually the info level is printed too much which is quite annoying.